packages/common/ecs.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const aws = require('./aws');
+
+/**
+ * list the clusters
+ *
+ * @returns {Array<string>} the cluster arns
+ */
+async function listEcsClusters() {
+  let response;
+  let clusterArns = [];
+  do {
+    const nextToken = response ? response.nextToken : null;
+    const params = { maxResults: 100, nextToken };
+    // eslint-disable-next-line no-await-in-loop
+    response = await aws.ecs().listClusters().promise(params);
+    clusterArns = clusterArns.concat(response.clusterArns);
+  } while (response.nextToken);
+
+  return clusterArns;
+}
+
+/**
+ * get cluster and service arns for the given stack and service task
+ *
+ * @param {string} stackName -  The name of the stack
+ * @param {string} taskName - The task name of ECS service
+ * @returns {Object} the object with `cluster` and `service` arns
+ */
+async function getEcsClusterService(stackName, taskName) {
```

Comment:
This function does two things:

1. Given a cluster name, return the cluster ARN
2. Given a cluster ARN and service name, return the ARN of that service

That should be broken into two separate functions.